BiometricLocker/BiometricLocker.swift

```diff
@@ -73,7 +85,7 @@ open class BiometricLocker {
     /// Locks the app.
     ///
     /// - Parameter date: Tells the locker when the app was sent to the background (or any other situation, when applicable), so that we can calculate if the app should be locked.
-    open func lock(at date: Date = Date()) {
+    public func lock(at date: Date = Date()) {
```

Comment:
This needs a better name. We don't actually lock by calling `lock()`. We just set the date so we can start timing, whether the app should lock itself up or not. Suggestions welcome.

Reply:
maybe `lockApp(after date: Date)`?

Reply:
But then we'd need the user to calculate when to lock. Right now you tell us when the action that should trigger a lockdown timeout takes place, and we take care of the maths. 🤔But then the current `lock(Date.distantPast)` thing is also quite bad. Will redo this!

Reply:
I meant keeping the default value, but yeah maybe `startLockdownTimeout(at:`?

Reply:
Better to abstract all this away. Have a method to lock either instantly or with the time allowance (default to 5min, can be set by the user). No more date nonsense 🚀